man/man8/zfs.8

```diff
@@ -38,7 +38,9 @@
 .Nd configures ZFS file systems
 .Sh SYNOPSIS
 .Nm
-.Fl ?
+.Fl ?V
+.Nm
+.Cm version
```

Comment:
nit: putting this at the beginning made sense when there wasn't a `zfs version` subcommand.  But since you've added one it should be placed in alphabetical order, which will be at the very end it seems.